src/utils/enum.ts

```diff
@@ -0,0 +1,8 @@
+export enum Grade {
+    A_PLUS = 'A+',
+    A_MINUS = 'A-',
+    B_PLUS = 'B+',
+    B_MINUS = 'B-',
+    F = 'F'
+  }
+  
```

Comment:
can you check why this sign is appearing here

Reply:
In EOF one space will causing this sign to show.